redoctober.go

```diff
@@ -242,6 +259,7 @@ func init() {
 	flag.StringVar(&hcKey, "hckey", "", "Hipchat API Key")
 	flag.StringVar(&hcRoom, "hcroom", "", "Hipchat Room Id")
 	flag.StringVar(&keysPath, "keys", "", "Path(s) of TLS private key in PEM format, comma-separated, must me in the same order as the certs")
+	flag.StringVar(&metricsPort, "metrics", "8081", "Port to use for metrics.")
```

Comment:
Any reason not to make the host configurable, too? What about machines with multiple external interfaces or service IPs?